kinetic/distribution.yaml

```diff
@@ -16140,6 +16140,16 @@ repositories:
       url: https://github.com/xaxxontech/xaxxon_openlidar.git
       version: master
     status: maintained
+  xbot_talker:
+    doc:
+      type: git
+      url: https://github.com/DroidAITech/xbot_talker.git
```

Comment:
The source repository doesn't have any license specified in either the package.xml, LICENSE file, or headers on the sources.  Please choose a license (https://opensource.org/licenses is a good place to start), and add it to the package.xml `<license>` tag and a LICENSE file in the root of the repository.